frontend/catalyst/jit.py

```diff
@@ -619,6 +647,28 @@ def f(x):
         former one.
 
 
+    .. details::
+        :title: Adding modules for Autograph conversion
+
+        Due to the experimental state of Autograph support, the following modules have been excluded
+        from Autograph conversion: ``pennylane, catalyst, jax``. However, it is possible to
+        enable these or other previously excluded modules via the ``autograph_include`` parameter,
+        which provides a list of modules that will always be evaluated before the default conversion
+        rules, which means that any module of this list will be enabled for conversion regardless of
+        the default conversion rules that were excluding it before.
```

Comment:
@rauletorresc I think this block wasn't updated with the new framing :)

Reply:
Somehow I missed it, thanks for checking it out. Updated now. Done.